packages/core/build/tokens.ts

```diff
@@ -6,43 +6,62 @@
 
 import { token, CdsTheme } from './token-utils';
 
+// internal optimization so tokens can refer to pre-calculated values rather than generating/duplicating the same calc over many tokens
+const internal = {
+  scale1: token('calc((1rem / var(--cds-global-base)) * var(--cds-global-scale-layout-space))', { raw: true }),
+  scale2: token('calc((1rem / var(--cds-global-base)) * var(--cds-global-scale-space))', { raw: true }),
+  scale3: token('calc((1rem / var(--cds-global-base)) * var(--cds-global-scale-typography))', { raw: true }),
+};
```

Comment:
This enables our spacing/sizing tokens to be converted to rem from the base font just like before but now adds a scale multiplier to easily adjust our three main sizing sets. This will make theming with space easier to adjust on the fly for future themes. (see the theme storybook below)